core/src/main/java/io/varhttp/VarConfiguration.java

```diff
@@ -38,6 +39,22 @@ public void addDefaultFilter(Class<? extends Filter> filter) {
 		context.addDefaultFilter(filter);
 	}
 
+	public void addDefaultVarFilter(Class<?> filterClass) {
+		context.addDefaultVarFilter(filterClass);
+	}
+
+	public void addDefaultVarFilter(Class<?> filterClass, Method method) {
```

Comment:
not sure if its a good idea to leave it expose but we can allow the two ways of configuration.